pkg/kubelet/winstats/perfcounters.go

```diff
@@ -0,0 +1,102 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package winstats
+
+import (
+	"errors"
+	"fmt"
+	"time"
+	"unsafe"
+
+	"github.com/lxn/win"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	cpuQuery                  = "\\Processor(_Total)\\% Processor Time"
+	memoryPrivWorkingSetQuery = "\\Process(_Total)\\Working Set - Private"
+	memoryCommittedBytesQuery = "\\Memory\\Committed Bytes"
+	perfCounterUpdatePeriod   = 1 * time.Second
```

Comment:
I choose 1 second because I believe it is the same as default [cadvisor collection period](https://github.com/google/cadvisor/blob/master/docs/runtime_options.md#housekeeping). I believe the operation is pretty lightweight based on my testing (kubelet process stays under 1% CPU on Windows) when idling. Do you think a different period would be better?

Reply:
I think it's fine but you should add a comment explaining why 1 second is chosen.

Reply:
Added a comment as you suggested.